.isort.cfg

```diff
@@ -3,6 +3,10 @@
 
 [settings]
 line_length=100
+multi_line_outddput=3
```

Comment:
This doesn't look like a valid setting?

Reply:
Correct, and that's why all these files are formatted wrong! https://github.com/psf/black#how-black-wraps-lines